raster/src/main/scala/geotrellis/raster/io/geotiff/GeoTiffMultibandTile.scala

```diff
@@ -537,6 +537,20 @@ abstract class GeoTiffMultibandTile(
    else iter.next._2
  }
 
+  /**
+   * Crop this tile to given pixel region of the given bands. The returned MultibandGeoTiffTile
+   * will contain a subset of bands that have the same area as the input GridBounds.
+   *
+   * @param bounds Pixel bounds specifying the crop area.
+   * @param  bandIndices       An array of band indexes.
+   *
+   */
+ def crop(bounds: GridBounds, bandIndices: Array[Int]): ArrayMultibandTile = {
+   val iter = crop(List(bounds), bandIndices)
+   if (iter.isEmpty) throw GeoAttrsError(s"No intersections of ${bounds} vs ${gridBounds}")
+   else iter.next._2
+ }
```

Comment:
We can DRY out this  code, by having the above crop call this overload. That's the preferred pattern through the  library.